test/input.txt

```diff
@@ -54,6 +54,13 @@
   # should not allow adding duplicate persons
   add Esther Potato p/555555 e/esther@not.a.real.potato pa/555, epsilon street t/tubers t/starchy
 
+##########################################################
+# test sort person command
+##########################################################
+
+  # addressbook should be sorted in alphabetical order ignoring case
```

Comment:
Should "add" one more Person with a name starting with any letter before 'E', before this "sort" test, so as to be able to test the "sort" function. Right now, the list is already sorted before the "sort" command is called.